Modules/Core/TestKernel/include/itkTestDriverInclude.h

```diff
@@ -73,7 +73,7 @@ std::map< std::string, int > RegressionTestBaselines(char *);
 using ComparePairType = std::pair< char *, char * >;
 
 // A structure to hold regression test parameters
-typedef struct
+using RegressionTestParameters = struct _RegressionTestParameters
```

Comment:
Why is additional name `_RegressionTestParameters` being introduced?

Reply:
These were needed to avoid an "anonymous struct" linking issue.  The name was needed in this case to be used ensure that the symbol has a name.